deepdrivemd/misc/OutlierDB.py

```diff
@@ -1,9 +1,10 @@
 import os
 import random
+from typing import List, Tuple  # noqa
 
 
 class OutlierDB:
-    def __init__(self, dir, restarts):
+    def __init__(self, dir: str, restarts: List):
```

Comment:
The list type should be written as List[type] where type may be str, int, another List[int], etc.

Reply:
Done